examples/monitor-prom-remotewrite-enable-basic-auth/basic-auth.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  username: YWRtaW4=
+  password: YWRtaW4=
+kind: Secret
+metadata:
+  name: basic-auth
+  namespace: default
```

Comment:
How about removing `namespace` from YAML files and using `-n` in commands instead?